packages/luv/build.sh

```diff
@@ -2,9 +2,9 @@ TERMUX_PKG_HOMEPAGE=https://github.com/luvit/luv
 TERMUX_PKG_DESCRIPTION="Bare libuv bindings for lua"
 TERMUX_PKG_LICENSE="Apache-2.0"
 TERMUX_PKG_MAINTAINER="@termux"
-TERMUX_PKG_VERSION="1.47.0-0"
-TERMUX_PKG_SRCURL=https://github.com/luvit/luv/releases/download/$TERMUX_PKG_VERSION/luv-$TERMUX_PKG_VERSION.tar.gz
-TERMUX_PKG_SHA256=2b5c460fad3b86ea7cfbfaa8bb284506e8f8a354bdd735bde5d664a7b2b1f8bd
+TERMUX_PKG_VERSION="1.48.0-0"
+TERMUX_PKG_SRCURL=https://github.com/luvit/luv/releases/download/v${TERMUX_PKG_VERSION}/luv-v${TERMUX_PKG_VERSION}.tar.gz
```

Comment:
This is the first release where I can see the 'v' suffix before version. Might be worth the effort asking the maintaintainers of luv to maintain consistent naming of tarballs

Reply:
@zhaozg would you like to share any thoughts about the of version style change?

Reply:
> @zhaozg would you like to share any thoughts about the of version style change?

I made https://github.com/luvit/luv/releases/tag/1.48.0-1

Reply:
Thank you. I have updated the tag here.